VotingApplication/VotingApplication.Web/Views/Create/ConfirmStep.cshtml

```diff
@@ -74,39 +73,43 @@
                 <md-list-item class="md-3-line">
 
                     <md-icon ng-if="expiryIsValid()" md-font-icon="fa-check" class="fa md-primary fa-2x" alt="check"></md-icon>
-                    <md-icon ng-if="!expiryIsValid()" md-font-icon="fa-exclamation-triangle" class="fa md-warn fa-2x" alt="exclamation triangle"></md-icon>
+                    <md-icon ng-if="!expiryIsValid()" md-font-icon="fa-exclamation-triangle" class="fa vo-warn-text fa-2x" alt="exclamation triangle"></md-icon>
 
                     <div class="md-list-item-text">
 
-                        <div ng-if=" newpoll.expiry">
-                            <h3>{{newPoll.Expiry}}</h3>
+                        <div ng-if="newPoll.DoesExpire">
+                            <h3>{{formatUtcPollExpiry()}}</h3>
                             <p>Your poll will expire and close. Voters won't be able to vote after this time</p>
                         </div>
 
-                        <div ng-if="!newPoll.Expiry">
+                        <div ng-if="!newPoll.DoesExpire">
                             <h3>Continuous poll</h3>
                             <p>Your poll won't expire.</p>
                         </div>
 
+                        <p ng-if="expiryDateIsInPast()" class="vo-warn-text">
+                            Your poll expiry date is in the past. This will cause it to immediately close.
+                        </p>
+
                     </div>
                     <md-divider></md-divider>
                 </md-list-item>
                 <md-list-item class="md-3-line">
 
                     <md-icon ng-if="invitationsAreValid()" md-font-icon="fa-check" class="fa md-primary fa-2x" alt="check"></md-icon>
-                    <md-icon ng-if="!invitationsAreValid()" md-font-icon="fa-exclamation-triangle" class="fa md-warn fa-2x" alt="exclamation triangle"></md-icon>
+                    <md-icon ng-if="!invitationsAreValid()" md-font-icon="fa-exclamation-triangle" class="fa vo-warn-text fa-2x" alt="exclamation triangle"></md-icon>
 
                     <div class="md-list-item-text">
-                        <h3>{{newPoll.Invitations.length}} Invitation(s)</h3>
+                        @*Maybe need to investigate how many times angular evaluates getValidInvitees().length*@
+                        <h3>{{getValidInvitees().length}} Invitation(s)</h3>
 
-                        <p ng-if="newPoll.Invitations.length">{{newPoll.Invitations.length}} invitee(s) will be emailed</p>
-                        <p ng-if="!newPoll.Invitations.length">No emails will be sent</p>
+                        <p ng-if="getValidInvitees().length > 0">{{getValidInvitees().length}} invitee(s) will be emailed</p>
```

Comment:
I'm wondering if we should be warning that not all emails are valid here, rather than just ignoring those that aren't and putting the onus on the user to work out that they entered x email addresses, but that we're only emailing y.